nodejs/week2/app.js

```diff
@@ -0,0 +1,50 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+const documents = require("./documents.json");
+
+// Support parsing JSON requests
+app.use(express.json());
+
+//GET /search
+app.get("/search", (req, res) => {
+  if (req.query.q) {
+    const containsQuery = documents.filter(function (entry) {
+      return entry.description.includes(req.query.q);
```

Comment:
This only searches the `description` field but the idea is to search all fields, at least the fields that contain strings.